syncano/models/fields.py

```diff
@@ -687,13 +686,52 @@ def to_native(self, value):
         return value
 
 
-GeoPointStruct = namedtuple('GeoPointHelper', ['latitude', 'longitude'])
+class GeoPoint(object):
```

Comment:
As it's not a field, it should go into a separate file in my opinion. structs.py or maybe even geo.py